src/shared/api/axios/index.ts

```diff
@@ -4,5 +4,4 @@ export const BASE_URL: string = import.meta.env.VITE_BASE_URL as string;
 
 export const axiosInstance = axios.create({
   baseURL: BASE_URL,
-  withCredentials: true,
```

Comment:
Why should u delete this line?

Reply:
because without it the calls for the backend doesn't work